src/emulate.c

```diff
@@ -665,30 +666,59 @@ static inline bool op_jal(struct riscv_t *rv, uint32_t insn)
     return false;
 }
 
+/* Get current time in microsecnds and update csr_time register */
+static inline void get_time_us(struct riscv_t *rv)
```

Comment:
Since the function is intentionally doing timekeeping, we can name it as `update_time`